server/src/controllers/user.controller.ts

```
@@ -31,7 +34,7 @@
     //set the token as an httpOnly cookie
     res.cookie('token', result, {
       httpOnly: true,
-      secure: true,
+      secure: process.env.NODE_ENV === 'production' ? true : false,
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)